src/GasEstimator.ts

```diff
@@ -0,0 +1,262 @@
+import {
+    constants,
+    ContractInteractor,
+    DeployTransactionRequest,
+    ERC20Token,
+    EstimateGasParams,
+    estimateMaxPossibleRelayCallWithLinearFit,
+    RelayTransactionRequest
+} from '@rsksmart/rif-relay-common';
+import {
+    DeployRequest,
+    DeployRequestStruct,
+    ForwardRequest,
+    RelayRequest
+} from '@rsksmart/rif-relay-contracts';
+import BigNumber from 'bignumber.js';
+import { toWei } from 'web3-utils';
+
+const SUBSIDY = BigNumber(12000);
```

Comment:
What does this mean and why is it a constant and not a configurable param?

Reply:
updated